setup.cfg

```diff
@@ -14,3 +14,7 @@ resources =
 [global]
 setup-hooks =
     pbr.hooks.setup_hook
+
+[entry_points]
+console_scripts =
+    ptero_workflow_execution_wrapper = ptero_workflow.scripts.execution_wrapper:main
```

Comment:
The presence of this entry point implies that you need to install the ptero-workflow service on every machine that's going to run workflow jobs.

I don't think this is something we want to do for a few reasons:

- it will cause us to alter the python environment that the job is run in (this is one of the current issues in flow)
- it might pigeon-hole us into shell-command-like services

To me, this seems like the kind of thing that probably belongs in the SDK rather than in the service.

-1